examples/base/README.md

```diff
@@ -0,0 +1,7 @@
+# RainCatcher BaseModule
+
+This is a template for modules in the RainCatcher project, it contains the default dotfiles and other configuration files for the development tooling and the directory structure spected from individual modules.
+
+RainCatcher code is written in [TypeScript](http://typescriptlang.org), uses [Mocha](https://mochajs.org/) for unit tests and [Istanbul](https://istanbul.js.org/) for test coverage checks.
+
+For an explanation of the expected internals of see the [main README](../../README.md#package-structure)
```

Comment:
I think that we missing actual point here:

>User need to copy paste this module to some other place/github repo etc. 
Manual changes to package.json are required. Let's also publish placeholder for this module to avoid someone using `@raincatcher/example-base` for their code :D